src/client/splitor.cpp

```diff
@@ -393,6 +401,54 @@ bool Splitor::MarkDiscardBitmap(IOTracker* iotracker, FileSegment* fileSegment,
     return true;
 }
 
+uint64_t Splitor::ProcessUnalignedRequests(const off_t currentOffset,
```

Comment:
If request is unaligned, need artificial alignment?

Reply:
yes, PaddingReadClosure will form an aligned request.